homeassistant/components/homekit_controller/manifest.json

```diff
@@ -3,7 +3,8 @@
   "name": "HomeKit Controller",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/homekit_controller",
-  "requirements": ["aiohomekit[IP]==0.2.41"],
+  "requirements": ["aiohomekit[IP]==0.2.43"],
```

Comment:
I think that you can drop `[IP]`.

From our installation logs:
```
 Requirement already satisfied: aiohomekit[IP]==0.2.41 in ./venv/lib/python3.7/site-packages (from -r requirements_all.txt (line 188)) (0.2.41)
  WARNING: aiohomekit 0.2.41 does not provide the extra 'ip'
```

Reply:
:+1: - it is originally supposed to have an [IP] feature flag, but it is getting all the deps Home Assistant needs without it and until add aio ble support to it we don't need the flag.